blog/2022-01-19-play-with-capact-via-cli.mdx

```diff
@@ -0,0 +1,334 @@
+        attributes:
+          - path: cap.attribute.capactio.examples.be-positive
+            revision: 0.1.0
+EOF
+```
+
+Next, create the Action with the Policy:
+
+```bash
+capact act create --name hello cap.interface.capactio.examples.greet --action-policy-from-file /tmp/policy.yaml
+```
+
+</TabItem>
+</Tabs>
+</div>
+
+2. Wait for the Action to have the `READY_TO_RUN` status:
+
+	```bash
+	capact act get hello
+	```
+3. When the status is `READY_TO_RUN`, run the Action:
+
+	```bash
+	capact act run hello
+	```
+4. Watch the progress:
+
+	```bash
+	capact act watch hello
+	```
+5. **(Optional)** Once the Action is finished, copy the name of the Pod for the `print` workflow step (column `PODNAME`). To read its logs, run:
+
+	```bash
+	kubectl logs {PODNAME} main
+	```
+  Example output:
+
+<div className="tab-container-nested">
+<Tabs
+className="disable-tabs"
+groupId="ref"
+defaultValue="def"
+values={[
+{label: 'Default Execution', value: 'def'},
+{label: 'Policy-Changed Execution', value: 'policy'},
+]}>
+<TabItem value="def">
+
+```bash
+ ____________________________
+< message: Hello from Capact >
+ ----------------------------
+    \
+     \
+      \
+                    ##        .
+              ## ## ##       ==
+           ## ## ## ##      ===
+       /""""""""""""""""___/ ===
+  ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
+       \______ o          __/
+        \    \        __/
+          \____\______/
+```
+
+</TabItem>
+<TabItem value="policy">
+
+```bash
+ __________________________________
+< message: Never gonna give you up >
+ ----------------------------------
+    \
+     \
+      \
+                    ##        .
+              ## ## ##       ==
+           ## ## ## ##      ===
+       /""""""""""""""""___/ ===
+  ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
+       \______ o          __/
+        \    \        __/
+          \____\______/
+```
+
+</TabItem>
+</Tabs>
+</div>
+
+6. Get the Action output (TypeInstance). Copy the `id` field value:
+
+	```bash
+	capact act get hello -ojson | jq '.Actions[0].output.typeInstances'
+	```
+
+	Example output:
+	```bash {3}
+  [
+    {
+      "id": "08fcaa07-7846-47af-b6a7-7c3818c69656",
+      "typeRef": {
+        "path": "cap.type.capactio.examples.message",
+        "revision": "0.1.0"
+      }
+    }
+  ]
+	```
+
+7. Get the TypeInstance `value.message` field value:
+
+   ```bash
+   capact ti get {ID} -ojson | jq -r '.[0].latestResourceVersion.spec.value.message'
+   ```
+
+8. Delete the Action:
+
+   ```bash
+   capact act delete hello
+   ```
+
+Let's consider another example, as life isn’t all sunshine and rainbows. Sometimes we need a more positively-colored message. Capact allows you to do it with ease ✨
+
+To choose a different Implementation for the `say` Interface, we need to prepare a dedicated Policy. The power of it is that you don't have to change the main workflow. You treat the step as a building block and just swap it for another existing one.
+
```

Comment:
I think that's a great place to put this GIF (or different one) 😄 
![tenor-139575536](https://user-images.githubusercontent.com/7155799/151161102-51298f89-5500-4b8e-ae1f-af7a62d59a5f.gif)

Reply:
good idea, I also like that one 👍 
Other ideas:
1. ![no-more-training](https://user-images.githubusercontent.com/17568639/151320790-e2734cec-2c85-417a-9bf0-38e0c289dde9.gif)
2. ![shocked-happy](https://user-images.githubusercontent.com/17568639/151320819-fd4ac198-7211-4039-a3c3-2e66a05b19e5.gif)

I also created dedicated ones:

3. <img src="https://user-images.githubusercontent.com/17568639/151320976-e74c6575-0698-47d0-afde-a43c597d62f1.gif" alt="drawing" width="450"/>

4. <img src="https://user-images.githubusercontent.com/17568639/151321017-0ca30635-8d16-4d56-80fa-60ea68bad7a7.gif" alt="drawing" width="450"/>


let's also for others to decide :)

Reply:
I like Star Wars meme #4